docs/config/README.md

```diff
@@ -2247,6 +2247,43 @@ python_binary = "python3"
 detect_extensions = []
 ```
 
+## R
+
+The `r` module shows the currently installed version of R.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `.R` file
```

Comment:
```suggestion
- The current directory contains a file with `.r` extension
```

Reply:
^ Purely for consistency as all other modules use a similar grammatical system like that

Reply:
Thank you for the suggestion.